my_compassion/static/src/js/show_password.js

```diff
@@ -1,32 +1,16 @@
-document.addEventListener("DOMContentLoaded", function (event) {
-  odoo.define("my_compassion.show_password", function (require) {
-    "use strict";
+document.addEventListener("DOMContentLoaded", function () {
+    const toggleBtn = document.querySelector('.toggle-password');
+    const passwordInput = document.getElementById('password');
 
-    function togglePasswordVisibility(inputId, icon) {
-      const input = document.getElementById(inputId);
-      if (input) {
-        if (input.type === "password") {
-          input.type = "text";
-          icon.classList.add("fa-eye-slash");
-          icon.classList.remove("fa-eye");
-        } else {
-          input.type = "password";
-          icon.classList.add("fa-eye");
-          icon.classList.remove("fa-eye-slash");
-        }
-      }
-    }
+    if (toggleBtn && passwordInput) {
+        toggleBtn.addEventListener('click', function () {
+            const isPasswordHidden = passwordInput.type === 'password';
+
+            passwordInput.type = isPasswordHidden ? 'text' : 'password';
 
-    function addPasswordToggleListener(eyeId, inputId) {
-      const eyeIcon = document.getElementById(eyeId);
-      if (eyeIcon) {
-        eyeIcon.addEventListener("click", function () {
-          togglePasswordVisibility(inputId, eyeIcon);
+            // Update icon correctly
+            toggleBtn.classList.toggle('fa-eye', isPasswordHidden);        // show eye when visible
+            toggleBtn.classList.toggle('fa-eye-slash', !isPasswordHidden); // show eye-slash when hidden
         });
-      }
     }
-
-    addPasswordToggleListener("eye_password", "password");
-    addPasswordToggleListener("eye_confirm_password", "confirm_password");
-  });
-});
+});
```

Comment:
The changes made broke the eye icon behavior in other login views. We need this functionality to handle all login pages